mqtt/aedes/package-lock.json

```diff
@@ -13,8 +13,8 @@
       "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.5.2.tgz",
       "integrity": "sha512-eBKD/FPxQoY1x6QONW2nBd54QUEyzcFP9FenujmoeDPy1rutVSHki1s/wR68F6O1QfCNDx+ayBH1O2CVNMzyyw==",
       "requires": {
-        "lodash.camelcase": "4.3.0",
-        "protobufjs": "6.8.8"
+        "lodash.camelcase": "^4.3.0",
```

Comment:
Not sure if those changes are made by mistake, but there is a lot of version changes. Can you please check ?